Userland/keymap.cpp

```diff
@@ -16,32 +16,33 @@ char* read_map(const JsonObject& json, const String& name)
     char* map = new char[80];
 
     auto map_arr = json.get(name).as_array();
-    for(int i=0; i<map_arr.size(); i++) 
+    for(int i=0; i<map_arr.size(); i++)
```

Comment:
Please add a space character after `for`.
You can use `clang-format` to automatically align everything with the project coding style.
(I have it bound to Alt+Shift+F in my editor. :)

Reply:
I'll reformat using `.clang-format` from project root tomorrow, seems my editor used some random style by default (which you probably don't want) ^^